ansible_collections/serverscom/sc_api/plugins/modules/sc_rbs_volume_credentials_info.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2025, Servers.com
+# GNU General Public License v3.0
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: sc_rbs_volume_credentials_info
```

Comment:
Do we need a separate module for this? I think, it should be part of the sc_rbs_volume_info.